draft-packages/form/KaizenDraft/Form/decision-tokens.scss

```diff
@@ -2,71 +2,80 @@
 @import "~@kaizen/design-tokens/sass/color-vars";
 
 // General purpose tokens
-$dt-color-form-border-color: var(
-  --kz-var-color-iron,
-  $kz-var-color-wisteria-500
-);
+$dt-color-form-border-color: var(--kz-var-color-iron, $kz-color-wisteria-500);
 $dt-color-form-border-color-focus: $kz-var-color-cluny-500;
 $dt-color-form-border-color-hover: var(
   --kz-var-color-slate,
-  $kz-var-color-wisteria-700
+  $kz-color-wisteria-700
 );
 $dt-color-form-border-color-reversed-focus: $kz-var-color-cluny-300;
 $dt-color-form-text-color: $kz-var-color-wisteria-800;
 $dt-color-form-text-color-reversed: $kz-var-color-white;
+$dt-color-form-text-color-label: rgba(
+  $kz-var-color-wisteria-800-rgb-params,
+  0.75
+);
+$dt-color-form-text-color-label-reversed: rgba(
+  $kz-var-color-white-rgb-params,
+  0.8
+);
 $dt-color-form-color-icon-reversed: var(
   --kz-var-color-slate,
-  $kz-var-color-wisteria-700
+  $kz-color-wisteria-700
+);
+$dt-color-form-text-color-placeholder: rgba(
+  $kz-var-color-wisteria-800-rgb-params,
+  0.7
 );
 
 // component specific tokens
 $dt-color-checkbox-background-color-checked: var(
   --kz-var-color-iron,
```

Comment:
If you feel like it you can use `@kaizen/design-tokens/sass/variable-identifiers.scss` :)

Reply:
Fixed now. 🤦‍♀️ It might be good to create a stylelint check to stop people from accessing the raw Kaizen css variables.